src/__swaps__/utils/numbers.ts

```diff
@@ -176,12 +177,10 @@ export const convertAmountToNativeAmount = (amount: BigNumberish, priceUnit: Big
 export const convertAmountAndPriceToNativeDisplay = (
```

Comment:
given that `convertAmountAndPriceToNativeDisplay` and `convertAmountAndPriceToNativeDisplayWithThreshold` are now basically the same with just the `useThreshold` arg being different, can we do the same and consolidate these or make a ticket to do this as clean up (ideally we would also convert the args to an object while we're at it)

Reply:
what's the policy on putting args in an object or not